code/modules/mob/living/simple_animal/slime/slime.dm

```diff
@@ -42,15 +42,17 @@
 
 	var/number = 0 // Used to understand when someone is talking to it
 
-	var/mob/living/Target = null // AI variable - tells the slime to hunt this down
-	var/mob/living/Leader = null // AI variable - tells the slime to follow this person
+	// Don't modify these directly. Use set_Target()/set_Leader()
+	VAR_PROTECTED/mob/living/Target = null // AI variable - tells the slime to hunt this down
+	VAR_PROTECTED/mob/living/Leader = null // AI variable - tells the slime to follow this person
```

Comment:
Hold up we can actually define private vars with sdmm?

Reply:
Yes, private and protected. See: https://github.com/ParadiseSS13/Paradise/blob/31e34af4ba88fe7e30147a122817409b1dc088df/code/__DEFINES/_spacemandmm.dm#L2-L14 for code or https://github.com/SpaceManiac/SpacemanDMM/tree/master/crates/dreamchecker#extensions for official docs